### PR TITLE
Navigation Bar

### PR DESCRIPTION
**Pull request by KaroFr**

### CODE REVIEW

**Comment by KaroFr** [comment]
[Einzelne Top-Elemente durch eine Material-Toolbar ersetzen](https://trello.com/c/VkzWSHNw/368-einzelne-top-elemente-durch-eine-material-toolbar-ersetzen)

File: projects/wave-core-new/src/lib/layers/layer-list/layer-list-menu/layer-list-menu.component.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import {Observable} from 'rxjs';
+import {Config} from '../../../config.service';
+
+@Component({
+    selector: 'wave-layer-list-menu',
+    templateUrl: './layer-list-menu.component.html',
+    styleUrls: ['./layer-list-menu.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class LayerListMenuComponent implements OnInit {
```
Comment (jdroenner) [inline projects/wave-core-new/src/lib/layers/layer-list/layer-list-menu/layer-list-menu.component.ts:14]: do you need onInit here?

File: projects/wave-core-new/src/lib/layers/layer-list/layer-list-menu/layer-list-menu.component.scss [hunk 2]
```diff
@@ -0,0 +1,46 @@
+.container {
+    max-height: inherit;
+    height: inherit;
+    border-right: solid 1px #ccc;
```
Comment (jdroenner) [inline projects/wave-core-new/src/lib/layers/layer-list/layer-list-menu/layer-list-menu.component.scss:4]: i  think this needs to go

File: projects/wave-core-new/src/lib/wave-core.module.ts [hunk 3]
```diff
@@ -237,7 +239,7 @@ const WAVE_COMPONENTS = [
 ];
 
 @NgModule({
-    declarations: [...WAVE_PIPES, ...WAVE_COMPONENTS],
+    declarations: [...WAVE_PIPES, ...WAVE_COMPONENTS, LayerListMenuComponent],
```
Comment (jdroenner) [inline projects/wave-core-new/src/lib/wave-core.module.ts:242]: LayerListMenuComponent is already in WAVE_COMPONENTS

**Review by jdroenner — CHANGES_REQUESTED** [review]
please look where the 1px white border comes from

File: projects/wave-core-new/src/lib/layers/layer-list/layer-list-menu/layer-list-menu.component.ts (hunk 1 above)
Comment (KaroFr) [reply projects/wave-core-new/src/lib/layers/layer-list/layer-list-menu/layer-list-menu.component.ts]: I removed it f23136c0e44778bc3c94661b388e236402d2f49f

File: projects/wave-core-new/src/lib/layers/layer-list/layer-list-menu/layer-list-menu.component.scss (hunk 2 above)
Comment (KaroFr) [reply projects/wave-core-new/src/lib/layers/layer-list/layer-list-menu/layer-list-menu.component.scss]: I removed the borders 49a729f5300f16b060c80af4725e1037e8d29681

File: projects/wave-core-new/src/lib/wave-core.module.ts (hunk 3 above)
Comment (KaroFr) [reply projects/wave-core-new/src/lib/wave-core.module.ts]: b187362c5e0fdae195ec8e2a0550f3af3c36cc66

**Review by jdroenner — DISMISSED** [review]

**Review by ChristianBeilschmidt — APPROVED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Pull request merged**